### PR TITLE
R for results

### PR DESCRIPTION
**Pull request by nazdir**
adds /ww r to get poll results

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -2,23 +2,21 @@ module.exports = {
...
   parserOptions: {
-    ecmaVersion: 2018,
+    ecmaVersion: 2018
   },
   rules: {
-    'vars-on-top': "off",
+    'vars-on-top': 'off',
     'no-case-declarations': 'off',
-    'radix': 'off',
+    radix: 'off',
```
Comment (ZachariBarnes) [inline .eslintrc.js]: why? This property becomes inconsistent with the others...

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "printWidth": 100,
+  "tabWidth": 2,
```
Comment (ZachariBarnes) [inline .prettierrc]: We're using 4 spaces in this project instead of 2

File: gameFunctions.js [hunk 3]
```diff
@@ -1,336 +1,370 @@
-require('dotenv').config();
+require('dotenv').config()
 
-const resFunc = require('./responseFunctions');
-const pollFuncs = require('./pollFunctions');
-const request = require('request');
-const logger = require('./logFunctions');
-const auth = require('./source/data/auth');
-const polls = require('./source/data/polls');
```
Comment (ZachariBarnes) [inline gameFunctions.js:8]: I prefer to keep the semi-colons.

File: source/data/polls.js [hunk 4]
```diff
@@ -1,107 +1,108 @@
...
-    return db.query('SELECT *, poll_votes.id as poll_vote_id, poll_options.id as poll_options_id FROM polls INNER JOIN poll_options ON polls.id = poll_options.poll_id LEFT JOIN poll_votes ON poll_options.id = poll_votes.option_id LEFT JOIN users ON poll_votes.voter_id = users.id where polls.team_id= $1 and polls.channel_id =$2 and polls.is_closed = $3',
-    [teamId, channelId, isClosed]).then((results) => {
-        return results.rows;
-    });
-};
+const getByChannelWithResults = async req => {
+  const { teamId, channelId, isClosed } = req
+  return db
+    .query(
+      'SELECT *, poll_votes.id as poll_vote_id, poll_options.id as poll_options_id FROM polls INNER JOIN poll_options ON polls.id = poll_options.poll_id LEFT JOIN poll_votes ON poll_options.id = poll_votes.option_id LEFT JOIN users ON poll_votes.voter_id = users.id where polls.team_id= $1 and polls.channel_id =$2 and polls.is_closed = $3',
```
Comment (ZachariBarnes) [inline source/data/polls.js]: This line definitely exceeds 100 characters, If you're fixing formatting please break up this line.

File: .prettierrc [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+  "useTabs": false,
+  "semi": false,
```
Comment (ZachariBarnes) [inline .prettierrc]: Please set semi to true

File: index.js [hunk 6]
```diff
@@ -1,187 +1,192 @@
...
+      if (commandArray.length) {
+        switch (commandArray[0]) {
+          case 'poll':
+            pollFuncs.createNewPoll(res, requestBody, commandArray)
+            break
+          case 'game':
+            gameFuncs.startNewGame(res, requestBody)
+            break
+          case 'results':
+          case 'r':
```
Comment (ZachariBarnes) [inline index.js]: can you swap the order of case 'r' and case 'results'so that its slightly more verbose as to what this functionality does?

**Review by ZachariBarnes — CHANGES_REQUESTED** [review]
The change is correct, but I am being a little nit-picky about formatting. Sorry

File: .eslintrc.js (hunk 1 above)
Comment (nazdir) [reply .eslintrc.js]: I'll save this without formatting to keep this from switching.

**Review by ZachariBarnes — APPROVED** [review]

**Pull request merged**